src/Stratis.Features.FederatedPeg/FederationGatewayFeature.cs

```diff
@@ -229,13 +228,22 @@ private string CollectStats()
                 benchLog.AppendLine();
             }
 
-            // Display recent withdrawals (if any).
-            // TODO: What order do these come out in?
-            List<WithdrawalModel> withdrawals = this.withdrawalHistoryProvider.GetHistory(TransfersToDisplay);
-            if (withdrawals.Count > 0)
+            List<WithdrawalModel> pendingWithdrawals = this.withdrawalHistoryProvider.GetPending();
+
+            if (pendingWithdrawals.Count > 0)
+            {
+                benchLog.AppendLine("--- Pending Withdrawals ---");
```

Comment:
Brilliant

Reply:
We do lose a bit of information this way (two lists) - i.e. how the different status transfers relate chronologically.